auto-voice-channels.py

```diff
@@ -20,13 +21,20 @@
 from functions import log, echo
 from discord.ext.tasks import loop
 
-logging.basicConfig(level=logging.INFO)
+#logging.basicConfig(level=logging.INFO)

```

Comment:
Was there something conflicting with docker logging or something? We should probably keep logging enable for all the discord.py log messages that help debug performance issues.